components-compose/src/main/java/uk/gov/hmrc/components/compose/organism/primary/PrimaryCardView.kt

```diff
@@ -15,4 +15,60 @@
  */
 package uk.gov.hmrc.components.compose.organism.primary
 
-class PrimaryCardView
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import uk.gov.hmrc.components.compose.atom.heading.Heading5
+import uk.gov.hmrc.components.compose.organism.HmrcCardView
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme
+
+@Composable
+fun PrimaryCardView(
+    modifier: Modifier = Modifier,
+    title: String,
+    titleContentDescription: String = "",
+    childPadding: Boolean = true,
+    content: @Composable (ColumnScope.() -> Unit)? = null
+) {
+    val childPaddingState by remember { derivedStateOf { childPadding } }
+
+    HmrcCardView(modifier = modifier) {
+        Heading5(
+            text = title,
+            modifier = Modifier
+                .padding(
+                    start = HmrcTheme.dimensions.hmrcSpacing16,
+                    end = HmrcTheme.dimensions.hmrcSpacing16,
+                    top = HmrcTheme.dimensions.hmrcSpacing16
+                )
+                .semantics(mergeDescendants = true) {
+                    contentDescription = titleContentDescription
+                }
+                .fillMaxWidth()
+        )
+        if (content != null) {
+            Column(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .then(
+                        if (childPaddingState) {
+                            Modifier
+                                .padding(HmrcTheme.dimensions.hmrcSpacing16)
+                        } else {
+                            Modifier
+                        }
+                    )
+            ) {
+                content()
+            }
+        }
+    }
+}
```

Comment:
Please can you add a couple of Preview functions to show the states with the minimum and the maximum? eg. one without any content and one with some content?